habushu-maven-plugin/src/test/resources/specifications/containerize-dependencies.feature

```diff
@@ -6,7 +6,16 @@ Feature: Test staging source files of specified dependencies and transitive path
     When the containerize-dependencies goal is executed
     Then the source files of the dependency and transitive Habushu-type dependencies are staged for containerization
 
-  Scenario: No Habushu-type dependencies are specified with the default configuration
-    Given no Habushu-type dependencies
```

Comment:
I: we no longer needs this scenario since we want to make sure user always insert the dependency